.dockerignore

```diff
@@ -0,0 +1,32 @@
+# virtual environments
```

Comment:
We may have to merge ideas from this PR into the other in general I say both PR's are good and merging the best of both worlds will be the best thing here.  Let's all have a sync tomorrow for the merger. This is the priority task IMHO!